stable/datacube-dashboard/values.yaml

```diff
@@ -21,15 +21,15 @@ resources:
   limits:
     cpu: "300m"
     memory: 1024Mi
-dockerArgs: [ "gunicorn", "-b", "0.0.0.0:8080", "-w", "1", "--timeout", "60", "cubedash:app" ]
+dockerArgs: [ "gunicorn", "-b", "0.0.0.0:8080", "-w", "4", "--timeout", "60", "cubedash:app" ]
 annotations:
-  iam.amazonaws.com/role: kubernetes-wms
+  iam.amazonaws.com/role: user.dev-eks-datacube
```

Comment:
You cannot set the user specific IAM roles check with the wms roles for EKS cluster

Reply:
Created a new IMA role `dev-eks-datacube-cubedash`